input/intro-notes/StructureDefinition-au-core-encounter-intro.md

```diff
@@ -23,13 +23,12 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [US Core
   - Although both are marked as must support, servers are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
   - A client application **SHALL** support both elements
 
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Encounter.type, as its inclusion in AU Core may be premature without clear overall support from existing systems. Furthermore, the associated preferred binding is unlikely to be supported by existing systems, suggesting that support for free text or local codes may be the most feasible approach at this point.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44580">FHIR-44580</a>.</p>
-
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Encounter.serviceType, as its inclusion in AU Core may be premature without clear overall support from existing systems. Furthermore, the associated preferred binding is unlikely to be supported by existing systems, suggesting that support for free text or local codes may be the most feasible approach at this point.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44581">FHIR-44581</a>.</p>
-
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove Must Support from Encounter.reasonReference, as its inclusion in AU Core may be premature without clear overall support from existing systems.<br/>Please comment on HL7 Jira <a href="https://jira.hl7.org/browse/FHIR-44582">FHIR-44582</a>.</p>
-
-
-
-
-
+<div class="request-for-feedback">
+    <p>Specific feedback is sought on a series of proposals to remove Must Support from the following elements:
```

Comment:
remove the following words across the files "a series of"

Reply:
Done